### ISSUE TITLE
Multiplayer Bugs (Weather Forecast Bug, Flickering Lag, and Lobby Bug)

### ISSUE THREAD

**Issue by Seimix** [Robosturm/Commander_Wars#365]
There are quite a few bugs I found in multiplayer that I feel like I should report here:

**1. Weather Forecast Bug**
After loading a saved Multiplayer game that has Random Weather & Forecast enabled, the forecast ends up breaking. It shows the wrong weather for player one and just doesn't show up for the other player. (If I need to, I can send you the save file)
NOTE: After playing the lobby for a bit longer, it seems to fix itself...

**2. Flickering Lag**
Now, this is a weird one...I don't know if this just has something to do with large maps, multiplayer, or both. But a kind of lag occurs when scrolling the camera or moving units, where it causes the camera and units to flicker. And of course, it gets worse the more units there are.
     Now, this one is especially curious to me. I feel like this sort of lag shouldn't be so prominent in multiplayer... and if it's not just a multiplayer bug, and is just a result of large maps and several units...is the engine just unable to handle all that?

**3. Lobby bug after going back to rules**
Basically, if you go back to the rules section in a multiplayer lobby after going to the CO Menu and returning back to the CO Menu, players can't join. And if you go back to the rules and go back to the CO menu after a player has joined, you can't start the match.

---------------
If I remember or find any more bugs regarding Multiplayer, I'll respond to this topic or edit this post.

**Comment by Robosturm** [comment]
Please report each bug as a seperate issue so i have the freedom to solve them seperatly.
The next time. Thx. :)

**Comment by JDOG30** [comment]
who are u talking to.

**Comment by Robosturm** [comment]
Not exactly sure. What you mean by that.
The best way to contact me is Discord. YouTube/Reddit etc are normally just shout outs.
And github is only used for priect tracking.
So keep issues project related

**Comment by JDOG30** [comment]
shut up
